### PR TITLE
Add pt-br/docs/reference/glossary/code-contributor.md

### PR DESCRIPTION
**Pull request by MrErlison**
Added translation for portuguese of the glossary term code-contributor.

### CODE REVIEW

File: content/pt-br/docs/reference/glossary/code-contributor.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+aka: 
+tags:
+- community
+- user-type
+---
+ Uma pessoa que desenvolve e contribui com código para o código aberto do Kubernetes.
+
+<!--more--> 
+
+Eles também são ativos {{< glossary_tooltip text="membros da comunidade" term_id="member" >}} que participam de um ou mais {{< glossary_tooltip text="Grupos de Interesse Especial (SIGs)" term_id="sig" >}}.
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/glossary/code-contributor.md:18]: Very minor: shouldn't "ativos" come after "membros da comunidade"?

Comment (MrErlison) [reply content/pt-br/docs/reference/glossary/code-contributor.md]: Ops! Sorry about that.

**Pull request merged**